UiPath.FreeRdpClient/UiPath.SessionTools.Tests/FakeProcessRunnerFixture.cs

```diff
@@ -0,0 +1,34 @@
+﻿using Moq;
+using System.Linq.Expressions;
+
+namespace UiPath.SessionTools.Tests;
+
+public class FakeProcessRunnerFixture 
```

Comment:
do you use this?

Reply:
thanks for pointing this out, I'm looking into the mess right now